src/components/Loader/Loader.test.tsx

```diff
@@ -1,18 +1,30 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import { BrowserRouter } from 'react-router-dom';
-
 import Loader from './Loader';
 
 describe('Testing Loader component', () => {
   test('Component should be rendered properly', () => {
-    render(
-      <BrowserRouter>
-        <Loader />
-      </BrowserRouter>
-    );
+    render(<Loader />);
 
-    expect(screen.getByTestId('spinner-wrapper')).toBeInTheDocument();
+    const spinnerWrapper = screen.getByTestId('spinner-wrapper');
+    expect(spinnerWrapper).toBeInTheDocument();
     expect(screen.getByTestId('spinner')).toBeInTheDocument();
+    expect(spinnerWrapper).toHaveClass('_spinner_wrapper_1vy2z_1');
```

Comment:
You should not include these classes which are generated on build time. Please remove this check.
```suggestion
```